uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-calendar-month.component.spec.js

```diff
@@ -473,9 +473,9 @@
             });
           });
 
-          describe('when the balance change of the leave request is positive', function () {
```

Comment:
That's correct @deb1990. Currently, only toil requests can have a positive balance_change